app/react/Metadata/components/MetadataFormFields.js

```diff
@@ -160,7 +161,11 @@ export class MetadataFormFields extends Component {
                       model={model}
                       field={`metadata.${property.name}`}
                     />
-                    {t(templateID, property.label)}
+                    {templateID ? (
```

Comment:
Is this if really necessary? are there scenarios where the template does not have _id?

Reply:
When multiple entities with different templates and common properties are edited, the template id is void and the translations module throws an exception